pkgs/development/compilers/dotnet/combine-packages.nix

```diff
@@ -3,18 +3,29 @@ packages:
 let cli = builtins.head packages;
 in
 assert lib.assertMsg ((builtins.length packages) != 0)
```

Comment:
Shouldn't this be `< 1` instead of `!= 0`? This doesn't make much sense to me.

1 package being allowed is also kind of odd to me, seeing how the entire point of this function is to combine multiple. I guess it might be convenient for quickly changing stuff.

Reply:
Honestly, I didn't touch this part at all, I just kept it as is to avoid breaking any existing usages of it.
I assume it might be useful if you're programmatically determining the list of packages to be combined and in one edge case it has only 1 and you don't want to special case it to return the package directly.